state.sh

```diff
@@ -16,75 +18,76 @@ if [ $# -eq 3 ]; then
     staker_lst=$(cast to-unit $x ether)
-    echo owner: $owner_lst LST && echo owner: $owner_zil ZIL unstaked
-    echo staker: $staker_lst LST && echo staker: $staker_zil ZIL unstaked
+    echo owner: $owner_lst $symbol && echo owner: $owner_zil ZIL unstaked
+    echo staker: $staker_lst $symbol && echo staker: $staker_zil ZIL unstaked
 else
-    x=$(cast call $1 "getDelegatedStake()(uint256)" --from $owner --block $block_num --rpc-url http://localhost:4201 | sed 's/\[[^]]*\]//g')
+    x=$(cast call $1 "getDelegatedStake()(uint256)" --from $owner --block $block_num --rpc-url $url | sed 's/\[[^]]*\]//g')
     owner_staked=$(cast to-unit $x ether)
-    x=$(cast call $1 "getDelegatedStake()(uint256)" --from $2 --block $block_num --rpc-url http://localhost:4201 | sed 's/\[[^]]*\]//g')
+    x=$(cast call $1 "getDelegatedStake()(uint256)" --from $2 --block $block_num --rpc-url $url | sed 's/\[[^]]*\]//g')
     staker_staked=$(cast to-unit $x ether)
     echo owner: $owner_staked ZIL staked && echo owner: $owner_zil ZIL unstaked
     echo staker: $staker_staked ZIL staked && echo staker: $staker_zil ZIL unstaked
 fi
 
-stake=$(cast call $1 "getStake()(uint256)" --block $block_num --rpc-url http://localhost:4201 | sed 's/\[[^]]*\]//g')
-commissionNumerator=$(cast call $1 "getCommissionNumerator()(uint256)" --block $block_num --rpc-url http://localhost:4201 | sed 's/\[[^]]*\]//g')
-denominator=$(cast call $1 "DENOMINATOR()(uint256)" --block $block_num --rpc-url http://localhost:4201 | sed 's/\[[^]]*\]//g')
+stake=$(cast call $1 "getStake()(uint256)" --block $block_num --rpc-url $url | sed 's/\[[^]]*\]//g')
+commissionNumerator=$(cast call $1 "getCommissionNumerator()(uint256)" --block $block_num --rpc-url $url | sed 's/\[[^]]*\]//g')
+denominator=$(cast call $1 "DENOMINATOR()(uint256)" --block $block_num --rpc-url $url | sed 's/\[[^]]*\]//g')
 if [[ "$variant" == "ILiquidDelegation" ]]; then
-    totalSupply=$(cast call $lst "totalSupply()(uint256)" --block $block_num --rpc-url http://localhost:4201 | sed 's/\[[^]]*\]//g')
+    totalSupply=$(cast call $lst "totalSupply()(uint256)" --block $block_num --rpc-url $url | sed 's/\[[^]]*\]//g')
     if [[ $totalSupply -ne 0 ]]; then
         price=$(bc -l <<< "scale=36; ($stake+$rewardsBeforeUnstaking-($rewardsBeforeUnstaking-$taxedRewardsBeforeUnstaking)*$commissionNumerator/$denominator)/$totalSupply")
     else
         price=$(bc -l <<< "scale=36; 1/1")
     fi
-    price0=$(cast call $1 "getPrice()(uint256)" --block $block_num --rpc-url http://localhost:4201 | sed 's/\[[^]]*\]//g')
-    echo LST supply: $(cast to-unit $totalSupply ether) ZIL
-    echo LST price: $price \~ $(cast to-unit $price0 ether)
-    echo staker LST value: $(bc -l <<< "scale=18; $staker_lst*$price") ZIL
+    price0=$(cast call $1 "getPrice()(uint256)" --block $block_num --rpc-url $url | sed 's/\[[^]]*\]//g')
+    echo $symbol supply: $(cast to-unit $totalSupply ether)
```

Comment:
ZIL removed from the end on purpose?

Reply:
yes, the LST symbol can be anything but not ZIL, it was incorrect before